.github/workflows/pr-correctness.yml

```diff
@@ -3,6 +3,7 @@ name: PR correctness
 on:
   pull_request:
     types: [ labeled, unlabeled, opened, synchronize, reopened ]
+  workflow_call:
```

Comment:
This looks nice :+1: 

I'm wondering if we can split things, like:

* `correctness.yml`, defining all steps with a parameter `iteration`
* both `pr_correctness.yml` and `cron_correctness.yml` calling `uses: .github/workflows/correctness.yml@main` with the right iteration parameter.

That split will allow other developers to use this :handshake: